pgd/urls.py

```diff
@@ -17,8 +19,53 @@
 
     # Uncomment the next line to enable the admin:
     #(r'^admin/(.*)', admin.site.root),
```

Comment:
The admin.site.root gave me an error when I tried uncommented that line. I've added a line (new way ?) for making admin accessible